.github/workflows/image_build_and_test.yaml

```diff
@@ -93,6 +93,12 @@ jobs:
         working-directory: ../compose-etl
         run: echo "{\"db_host\":\"host.docker.internal\",\"db_username\":\"postgres\",\"db_password\":\"postgres\",\"db_database\":\"metadata_db\"}" > configs/creds.json
 
+      - name: Install docker-compose
+        working-directory: ../compose-etl
+        run: |
+          curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
that is a pretty old version of docker compose, do we want to use this change to get to a newer version?